ooni/ui/web/web.py

```diff
@@ -6,6 +6,11 @@
 from ooni.settings import config
 
 class WebUIService(service.MultiService):
+    """Contain any element to manage the web user interface service.
+
+
+    The class above inherits to another class wich is a services container."""
```

Comment:
Maybe it's sufficient to say. "This multiservice contains the ooniprobe web user interface"